registration/src/main/java/org/spine3/samples/lobby/registration/seat/availability/SeatsAvailabilityAggregate.java

```diff
@@ -44,7 +46,7 @@
  * @author Alexander Litus
  */
 @SuppressWarnings({"TypeMayBeWeakened"/** "OrBuilder" parameters are not applicable*/, "OverlyCoupledClass"})
-public class SeatsAvailabilityAggregate extends Aggregate<SeatsAvailabilityId, SeatsAvailability, SeatsAvailability.Builder> {
+public class SeatsAvailabilityAggregate extends AbstractLobbyAggregate<SeatsAvailabilityId, SeatsAvailability, SeatsAvailability.Builder> {
```

Comment:
Please make sure you follow 120-symbols-per-row restriction for this class.

Some parts seem to not, and thus should be split into either multiple lines or rewritten with the extracted variables instead of in-place method calls.